applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_k0_procedure_process.cpp

```diff
@@ -483,15 +480,9 @@ KRATOS_TEST_CASE_IN_SUITE(K0ProcedureChecksIfProcessHasCorrectMaterialData, Krat
     p_element->GetProperties().Erase(K0_VALUE_ZZ);
 
     p_element->GetProperties().SetValue(INDEX_OF_UMAT_PHI_PARAMETER, 1);
-    p_element->GetProperties().SetValue(NUMBER_OF_UMAT_PARAMETERS, 0);
     Vector umat_parameters{1};
     umat_parameters[0] = -30.0;
     p_element->GetProperties().SetValue(UMAT_PARAMETERS, umat_parameters);
-    KRATOS_EXPECT_EXCEPTION_IS_THROWN(process.Check(),
-                                      "INDEX_OF_UMAT_PHI_PARAMETER (1) is not in range 1, "
-                                      "NUMBER_OF_UMAT_PARAMETERS (0) for element 1.")
-
-    p_element->GetProperties().SetValue(NUMBER_OF_UMAT_PARAMETERS, 1);
```

Comment:
Shouldn't we replace this by a check where `INDEX_OF_UMAT_PHI_PARAMETER` is out of bounds? For instance, if we set it here to 2, then it should raise an error, since the size of `UMAT_PARAMETERS` is exceeded.